.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
```

Comment:
Kindly remove the `stickler` file
 
**NOTE:** Microverse has moved on to Github actions for continuous deployment please use that instead of stickler. 
![image](https://user-images.githubusercontent.com/31897434/85197008-14353c00-b2d6-11ea-932b-d3a0ab291c2a.png)

https://github.com/microverseinc/linters-config/tree/master/react-redux